### ISSUE TITLE
Various Accept types appear to return different results

### ISSUE THREAD

**Issue by ebremer** [ORCID/ORCID-Source#6804]
Comparing the result MIME types for the same URL:
text/turtle
application/ld+json
application/rdf+xml
application/n-triples

all return effectively different results.  JSON-LD seems to have the most, while turtle, n-triples, RDF/XML return data missing many of the same elements as JSON-LD such as affiliation data.  JSON-LD is the only one containing the Schema.org name space.

**Comment by TomDemeranville** [comment]
You're right about our JSON-LD, it uses a different namespace from the other endpoints.  

The turtle/rdf data was put in place many years ago by the community.  One of the few open source contributions we've had.  A few years later we added the JSON-LD response to contain more information based on the schema.org namespace.  

We didn't update the other LOD formats as they were being used by applications, although none of the LOD formats are used much compared to our native formats.

**Comment by TomDemeranville** [comment]
Out documentation on this is here: https://info.orcid.org/ufaqs/does-orcid-support-schema-org-linked-open-data-and-json-ld/
